src/utils/viewer/viewer.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Habitat WebGL Viewer</title>
+  <link rel="stylesheet" href="WebApplication.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+
+<body>
+  <h1>Habitat WebGL Viewer</h1>
+  <div id="container">
```

Comment:
Adding `class="fullsize"` will make it [occupy the whole browser window](https://doc.magnum.graphics/magnum/platforms-html5.html#platforms-html5-layout).